src/game/UIStateBar.cpp

```diff
@@ -245,11 +295,45 @@ void CUIStateBar::UpdatePosition(const __Vector3 & vPos, float fYaw) {
     m_fYawPlayer = fYaw;
 }
 
+void CUIStateBar::UpdateFPS(float fps) {
+    if (NULL == m_pText_Fps) {
+        return;
+    }
+
+    char szPos[64];
+    sprintf(szPos, "%.0f", fps);
+    m_pText_Fps->SetString(szPos);
+}
+
+void CUIStateBar::UpdateSystemTime() {
+    if (NULL == m_pText_SystemTime) {
+        return;
+    }
+
+    time_t      timer;
+    char        buffer[9];
+    struct tm * tm_info;
+
+    timer = time(NULL);
+    tm_info = localtime(&timer);
+
+    strftime(buffer, 9, "%H:%M:%S", tm_info);
+    m_pText_SystemTime->SetString(buffer);
+}
```

Comment:
We can remove this function and inline it in Tick as it's implemented in the official client:
```suggestion
char szBuff[12]{};
sprintf(szBuff, "%.1f", CN3Base::TimeGet());
m_pText_SystemTime->SetString(szBuff);
```

Also note that parsing chat command can toggle its visibility, hence we need a member variable m_bShowSystemTime.